systems/controllers/osc/operational_space_control.cc

```diff
@@ -260,17 +260,17 @@ void OperationalSpaceControl::Build() {
   // Add costs
   // 1. input cost
   if (W_input_.size() > 0) {
-    prog_->AddQuadraticCost(MatrixXd::Identity(n_u_, n_u_),
+    prog_->AddQuadraticCost(W_input_,
```

Comment:
Can't believe there is such a big bug, since the robot is balancing and walking in simulation!